examples/xa-transactions/xa_transactions.description

```diff
@@ -1,3 +1,4 @@
 // Ballerina XA transactions are used when the transaction is happening over
-// two or more databases or JMS endpoints. This sample uses two H2 DBs. Before
-// running the sample, change the DB connection properties as required.
+// two or more databases or JMS endpoints. This sample uses two H2 DBs (these
```

Comment:
No need to create databases before running the sample. Those get created within the sample itself.